src/Bowling.js

```diff
@@ -0,0 +1,104 @@
+var Scorecard = function() {
```

Comment:
If the file is called Bowling, the function should probably be called Bowling too (or, call the file Scorecard)